### DISCUSSION TITLE
Smallrye-config interface objects not working?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [mapstruct/mapstruct#3444] [Q&A] [answered]
Is there a way to get smallrye-config interfaces to work with mapstruct?
https://smallrye.io/smallrye-config/3.4.1/config/mappings/#nested-groups
My current solution is to implement a custom mapper and use an `expression` to map it.

**Comment by filiphr** [comment, marked as answer]
Best solution from what I can see is to use a custom `AccessorNamingStrategy` and return the methods as getters.
